project/Build.scala

```diff
@@ -2826,15 +2826,17 @@ object Build {
       BuildInfoPlugin.buildInfoDefaultSettings
 
   lazy val presentationCompilerSettings = {
-    val mtagsVersion = "1.6.2"
+    val mtagsVersion = "1.6.2+105-e665821a-SNAPSHOT"
```

Comment:
We cannot use a snapshot version, since now they get removed after 60 days. We need to wait for Metals release, so this will go into 3.8.1 most likely

Reply:
I think this will be next week because of all the other stuff going on

Reply:
Looking at the contents, I'd try to get this in 3.8.0, still... let's try to coordinate with @WojciechMazur and perhaps we could get it in time for 3.8.0-RC1?